### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by somexpert**
Main points:
1. Put model registrations in admin.py
2. Remove site_views.py and empty views.py
3. Get the code to comply with PEP-8 standards
4. Update our urls.py to use a better format
5. Replace our use of render_to_response with just render, since we always pass the request context

### CODE REVIEW

**Comment by somexpert** [comment]
@ldko, @damonkelley, this is ready for review.

File: accession/models.py [hunk 1]
```diff
@@ -107,7 +114,10 @@ class Donor(models.Model):
     middle_name = models.CharField(blank=True, max_length=100)
     last_name = models.CharField(blank=True, max_length=100)
     organization_name = models.CharField(blank=True, max_length=255)
-    donor_type = models.CharField(max_length=3, blank=True, choices=DONOR_TYPE_CHOICES, default="PER")
+    donor_type = models.CharField(max_length=3,
+                                  blank=True,
+                                  choices=DONOR_TYPE_CHOICES,
+                                  default="PER")
```
Comment (damonkelley) [inline accession/models.py]: This formatting is different from all the others. This is not wrong, nor do you need to change it. I just wanted to make sure you are aware of that.

Comment (somexpert) [inline accession/models.py]: Yeah, it sticks out. I'll fix it to be consistent with the rest.

File: accession/admin.py [hunk 2]
```diff
@@ -1,68 +1,131 @@
...
-    search_fields = ['object_number','object_description', 'remarks', 'public_notes', 'label__label']
+    list_display = ['object_number', 'object_description', 'showLabel',
+                    'showLocation', 'showDesigner']
+    search_fields = ['object_number', 'object_description', 'remarks',
+                     'public_notes', 'label__label']
+
 
 class CityAdmin(admin.ModelAdmin):
-    search_fields = ['city']
+    search_fields = ['city', ]
```
Comment (damonkelley) [inline accession/admin.py]: As discussed, we can actually just remove the comma from the single element lists.

File: accession/admin.py [hunk 3]
```diff
@@ -1,68 +1,131 @@
...
+            'fields': ('address_1', 'address_2', 'city', 'state',
+                       'postal_code', 'country')
         }),
         ('Contact Info', {
-            'fields': ('phone_number_1', 'phone_number_2', 'email_address'),
+            'fields': ('phone_number_1', 'phone_number_2', 'email_address')
         }),
         ('Comments', {
-            'fields': ('comments',),
+            'fields': ('comments', )
```
Comment (damonkelley) [inline accession/admin.py]: The extra space after the comma is not needed here either.

**Comment by damonkelley** [comment]
:+1:

**Comment by ldko** [comment]
Not specific to your changes in this PR, but do we need to add zip_safe=False to the setup function to enable running migrations when not running in Docker?

**Comment by somexpert** [comment]
Doesn't appear that it is needed, as I was able to install from GitHub into a brand new django project in a virtual environment and run the migration without any issue. However, in doing so, I did find that I need to add the django-queryset-csv dependency to setup.py.

**Comment by ldko** [comment]
To note as discussed elsewhere, `zip_safe=False` may be needed for `python setup.py install` but not `pip install`. We will put it in another PR with a couple of other small things. :+1:

**Pull request merged**